data_rentgen/server/api/v1/router/auth.py

```diff
@@ -51,3 +54,16 @@ async def auth_callback(
     request.session["refresh_token"] = code_grant["refresh_token"]
 
     return {}
+
+
+@router.get("/logout")
+async def logout(
+    request: Request,
+    current_user: Annotated[User, Depends(get_user())],
+    auth_provider: Annotated[KeycloakAuthProvider, Depends(Stub(AuthProvider))],
+):
+    refresh_token = request.session.get("refresh_token", None)
```

Comment:
Does it even work without refresh_token?

Reply:
No but I decide that will be better if we catch keycloak error response, then making our own.
But I guess it's good to decide should we remove token's from cookies anyway?